src/Control/Monad/Codensity.hs

```diff
@@ -247,27 +234,3 @@ instance (m ~~ m', MonadReader r m')
 improve :: Functor f => (forall m. MonadFree f m => m a) -> Free f a
 improve m = lowerCodensity m
 {-# INLINE improve #-}
-
-
--- | Wrap the remainder of the 'Codensity' action using the given
--- function.
---
--- This function can be used to register cleanup actions that will be
--- executed at the end.  Example:
---
--- > wrapCodensity (`finally` putStrLn "Done.")
-wrapCodensity :: (forall a. m a -> m a) -> Codensity m ()
-wrapCodensity f = Codensity (\k -> f (k ()))
-
--- | @'reset' m@ delimits the continuation of any 'shift' inside @m@.
---
--- * @'reset' ('return' m) = 'return' m@
-reset :: Monad m => Codensity m a -> Codensity m a
-reset = lift . lowerCodensity
-
--- | @'shift' f@ captures the continuation up to the nearest enclosing
--- 'reset' and passes it to @f@:
---
--- * @'reset' ('shift' f >>= k) = 'reset' (f ('lowerCodensity' . k))@
-shift :: Applicative m => (forall b. (a -> m b) -> Codensity m b) -> Codensity m a
-shift f = Codensity $ lowerCodensity . f
```

Comment:
Why are these functions now removed? This appears wholly unrelated to the `oneShot` changes.